packages/loader/container-loader/src/container.ts

```diff
@@ -1750,10 +1750,11 @@ export class Container
 
 		// Internal context is fully loaded at this point
 		this.setLoaded();
+		timings.end = performance.now();
 		this.subLogger.sendTelemetryEvent(
 			{
 				eventName: "LoadStagesTimings",
-				...timings,
+				details: JSON.stringify(timings),
```

Comment:
why are you storing the timestamp instead of differences ?

Reply:
We would need the difference between different events. like stage 2 minus obtain snapshot duration etc which we can do during the kusto queries.